src/walletConnect/screens/SessionRequest.tsx

```diff
@@ -86,66 +48,45 @@ function SessionRequest({ navigation, route: { params } }: Props) {
         e.preventDefault()
         deny()
       }),
-    [navigation, params, isLoading]
+    [navigation, session, isLoading]
   )
 
-  const { url, name, icon, methods } = getRequestInfo(params)
-  const fallbackIcon = icon ?? `${url}/favicon.ico`
+  const { url, name, icons } = session.params[0].peerMeta
+  const fallbackIcon = icons[0] ?? `${url}/favicon.ico`
 
   return (
-    <SafeAreaView style={styles.screen}>
-      <ScrollView contentContainerStyle={styles.scrollContainer}>
-        <View>
-          <View style={styles.center}>
-            <Image style={styles.logo} source={{ uri: fallbackIcon }} />
-          </View>
-          <Text style={styles.header} testID="SessionRequestHeader">
-            {t('connectToWallet', { dappName: name })}
-          </Text>
-
-          {methods.length > 0 && <Text style={styles.subHeader}>{t('sessionInfo')}</Text>}
+    <>
+      <View>
+        <View style={styles.center}>
+          <Image style={styles.logo} source={{ uri: fallbackIcon }} />
         </View>
-
-        {methods.length > 0 && (
```

Comment:
i removed this because `getRequestInfo` was returning `methods` as always `[]` so this block never executes